src/headers.rs

```diff
@@ -14,6 +14,7 @@ use hyper::header::{Header, HeaderFormat, parsing};
 ///
 /// This is an internal type not meant for general public usage, as its implementation limited in
```

Comment:
There still might be value in upstreaming this to hyper since servers may have to parse header this as well. cc @seanmonstar